### ISSUE TITLE
Ensure OpenGL_GL_PREFERENCE is also passed to GLEW project

### ISSUE THREAD

**Issue by jcfr** [BrownBiomechanics/Autoscoper#193]
To follow up on the SlicerAutoscoperM[^1] and Autoscoper[^2] updates, the GLEW External project also need to be updated to fix the following warning:

[^1]: https://github.com/BrownBiomechanics/SlicerAutoscoperM/pull/13
[^2]: https://github.com/BrownBiomechanics/Autoscoper/pull/111

```
[  7%] Performing configure step for 'GLEW'
loading initial cache file /work/Preview/S-0-E-b/SlicerAutoscoperM-build/Autoscoper-build/GLEW-prefix/tmp/GLEW-cache-Release.cmake
CMake Warning (dev) at /usr/share/cmake-3.22/Modules/FindOpenGL.cmake:315 (message):
  Policy CMP0072 is not set: FindOpenGL prefers GLVND by default when
  available.  Run "cmake --help-policy CMP0072" for policy details.  Use the
  cmake_policy command to set the policy and suppress this warning.

  FindOpenGL found both a legacy GL library:

    OPENGL_gl_LIBRARY: /usr/lib64/libGL.so

  and GLVND libraries for OpenGL and GLX:

    OPENGL_opengl_LIBRARY: /usr/lib64/libOpenGL.so
    OPENGL_glx_LIBRARY: /usr/lib64/libGLX.so

  OpenGL_GL_PREFERENCE has not been set to "GLVND" or "LEGACY", so for
  compatibility with CMake 3.10 and below the legacy GL library will be used.
Call Stack (most recent call first):
  CMakeLists.txt:43 (find_package)
This warning is for project developers.  Use -Wno-dev to suppress it.
```

References:
* https://github.com/BrownBiomechanics/SlicerAutoscoperM/pull/13
* https://github.com/BrownBiomechanics/Autoscoper/pull/111
